OPERATING.md

```diff
@@ -40,7 +40,7 @@ The adaptor requires a Mongodb-compatible database to manage its internal state.
 | GP2GP_MONGO_PASSWORD            |                           | Mongo database password. Leave undefined if GP2GP_MONGO_URI is used.                                                                                           |
 | GP2GP_MONGO_OPTIONS             |                           | Mongodb URL encoded parameters for the connection string without a leading "?". Leave undefined if GP2GP_MONGO_URI is used.                                    |
 | GP2GP_MONGO_AUTO_INDEX_CREATION | true                      | (Optional) Should auto index for Mongo database be created.                                                                                                    |
-| GP2GP_MONGO_TTL                 | P7D                       | (Optional) Time-to-live value for inbound and outbound state collection documents as an [ISO 8601 Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations). |
+| GP2GP_MONGO_TTL                 | P84D                      | (Optional) Time-to-live value for inbound and outbound state collection documents as an [ISO 8601 Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations). |
```

Comment:
The other option to P84D would be P12W, to say 12 weeks. 
If there is a preference for P12W, I can do that instead. Happy to receive advice.